src/app/dashboard/whitelist/WhiteListPage.js

```diff
@@ -0,0 +1,378 @@
+//@flow
+
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import DocumentTitle from 'react-document-title'
+import uuidv4 from 'uuid/v4'
+import {
+  DataTable,
+  PaginationV2,
+  Modal,
+  ModalWrapper,
+  DatePicker,
+  DatePickerInput,
+  FileUploaderButton,
+} from "carbon-components-react"
```

Comment:
Double quotes are deprecated. Probably we are missing this rule in our eslint-plugin. Could you add it there please?

Reply:
fixed 

https://github.com/PolymathNetwork/eslint-plugin-polymath-react/pull/1 made a PR here to add this rule